WordPress/UITests/Utils/XCTest+Extensions.swift

```diff
@@ -3,14 +3,22 @@ import XCTest
 
 extension XCTestCase {
 
-    public func setUpTestSuite(for app: XCUIApplication = XCUIApplication(), removeBeforeLaunching: Bool = false) {
+    public func setUpTestSuite(
+        for app: XCUIApplication = XCUIApplication(),
+        removeBeforeLaunching: Bool = false,
+        crashOnCoreDataConcurrencyIssues: Bool = true
+    ) {
         super.setUp()
 
         // In UI tests it is usually best to stop immediately when a failure occurs.
         continueAfterFailure = false
 
         app.launchArguments = ["-wpcom-api-base-url", WireMock.URL().absoluteString, "-no-animations", "-ui-testing"]
 
+        if crashOnCoreDataConcurrencyIssues {
+            app.launchArguments.append(contentsOf: ["-com.apple.CoreData.ConcurrencyDebug", "1"])
+        }
```

Comment:
Only a subset of the tests use `setUpTestSuite`, which would result in this argument being applied inconsistently.

Have you considered using the test plan to set it and use this method to opt-out only?

<img width="918" alt="image" src="https://user-images.githubusercontent.com/1218433/227426670-e8c852d7-e2e2-465b-8134-f1b202275010.png">

Reply:
Good suggestion. I hadn't thought about it, but I'll give it a go.

Reply:
> Only a subset of the tests use `setUpTestSuite`

The idea is only turn the debug option on for _UI tests_, which all calls this funciton, I think?

Reply:
> [UI tests] all calls this funciton, I think?

Oh, that is correct. I don't know why I got the impression only some of them called `setUpTestSuite` 🤔 But I checked each test and they all do.

Apologies.